assets/javascripts/birdeck.js

```diff
@@ -21,9 +21,7 @@ function renderPost(post) {
 }
 
 function fetchPostsButton() {
-  $("#button-fetch-posts").on("click", function(){
-    fetchPosts()
-    })
+  $("#button-fetch-posts").on("click", fetchPosts)
```

Comment:
@stevekinney Don't like this. It's not about being extraneous but about maintaining the same level of abstraction. In this case, it's having functions as the public interface of the implementation.